.yamato/_projectcontext.yml

```diff
@@ -388,8 +388,6 @@ all_package_ci_project_fast-trunk:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  upm-ci package izon -t
       -  upm-ci package izon -d
-    triggers:
-        expression: pull_request.target eq "master" AND NOT pull_request.draft AND pull_request.push.changes.any match ["**/Documentation*/**/*"]
```

Comment:
did this mean to get removed?

Reply:
Weird. Looking into it, thanks.

Reply:
It seems I made a mistake in a previous PR (https://github.com/Unity-Technologies/Graphics/pull/1304/files).
I prefixed {target_editor} with 'fast-' in the python builder, but when the target_editor itself is already `fast-trunk`, it can't match the expression, leading to these 2 lines not being generated.

Reply:
Fixed with the last commit but this will need a backport, as this line also exists on other branches.